src/states/adult.rs

```diff
@@ -235,6 +251,25 @@ impl Adult {
         );
         self.disconnect(&pub_id);
     }
+
+    fn handle_elders_update(
+        &mut self,
+        elders_info: EldersInfo,
+        proofs: AccumulatingProof,
+        gen_pfx_info: GenesisPfxInfo,
+        outbox: &mut dyn EventBox,
+    ) -> Result<Transition, RoutingError> {
+        self.gen_pfx_info = gen_pfx_info;
+        self.parsec_map = ParsecMap::new(self.full_id.clone(), &self.gen_pfx_info);
+        self.chain.add_elders_info(elders_info.clone(), proofs)?;
+        self.handle_section_info_event(
```

Comment:
Probably should not be needed as discussed as Adults become Elder through parsec